<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import File</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>-2</x>
     <y>0</y>
     <width>651</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>5</number>
    </property>
    <property name="topMargin" >
     <number>5</number>
    </property>
    <property name="rightMargin" >
     <number>5</number>
    </property>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Import File:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3" >
     <widget class="QLineEdit" name="importFileNameBox" />
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Protein Acronym:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="proteinAcronymBox" >
      <property name="maxLength" >
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>25</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>335</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Tilt Angle:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QSpinBox" name="tiltAngleBox" >
      <property name="maximum" >
       <number>90</number>
      </property>
      <property name="singleStep" >
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>380</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text" >
       <string>Frame#:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2" >
     <widget class="QLineEdit" name="frameNumberBox" >
      <property name="maxLength" >
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>335</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Sub-Image:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QLineEdit" name="subImageBox" >
      <property name="maxLength" >
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>380</width>
        <height>21</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>importFileNameBox</tabstop>
  <tabstop>proteinAcronymBox</tabstop>
  <tabstop>tiltAngleBox</tabstop>
  <tabstop>frameNumberBox</tabstop>
  <tabstop>subImageBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
